### DISCUSSION TITLE
Config Documentation

### DISCUSSION THREAD

**Discussion by caco3** [jomjol/AI-on-the-edge-device#1973] [General]
@jomjol @haverland 

While working on the Parameter Documentation I fond some oddities:
- [ ] The `Analogue section` misses the `CNNGoodThreshold` parameter in the UI. is this on purpose?
- [ ] The GPIO  4 and 12 are only partially `Expert parameters`. How should we treat this? 
- [ ] Is it right that `ExtendedResolution` in the `Analogue section` is deprecated?
- [ ] Is it right that now `AllowNegativeRates` is only available as a part of a number (but not standalone)? The Config page shows it as part of a Number, but in the config file it is generic. See https://github.com/jomjol/AI-on-the-edge-device/pull/1972/files
- [ ] `RSSIThreashold` seems to be missing in the config file. What should be the default value?

See also https://github.com/jomjol/AI-on-the-edge-device/pull/1972

**Comment by jomjol** [comment]
>     * [ ]  The `Analogue section` misses the `CNNGoodThreshold` parameter in the UI. is this on purpose?
This is coming from the fact, that at first there where only `ana-cont` type networks for them this parameter does not make sense.
@haverland : do you want to integrate it for `ana-class100` types?
> 
>     * [ ]  The GPIO  4 and 12 are only partially `Expert parameters`. How should we treat this?
> 
>     * [ ]  Is it right that `ExtendedResolution` in the `Analogue section` is deprecated?
Yes - but might be needed for downward compatiblity, if you don't set it in the Postprocessing section
> 
>     * [ ]  Is it right that now `AllowNegativeRates` is only available as a part of a number (but not standalone)? The Config page shows it as part of a Number, but in the config file it is generic. See https://github.com/jomjol/AI-on-the-edge-device/pull/1972/files
Yes - but also a topic of backward compatility. But once opening the config.ini and resaving would solve this problem.
> 
>     * [ ]  `RSSIThreashold` seems to be missing in the config file. What should be the default value?
This is experimental and even not for the expert - at least up to now. We can change that.

**Reply by ghost** [reply]
CNNGoodThreshold makes no sense for class-100. Because the loss function is not softmax. It uses logits: https://www.youtube.com/watch?v=Qn4Fme1fK-M 
I think where is not need for it.

**Reply by caco3** [reply]
So, to understand it right:
 - `CNNGoodThreshold` only makes sense for the following models `dig-class11`, `dig-cont`, `ana-cont`?
 - `CNNGoodThreshold` should also be shown for the analo gsection?
